CMakeLists.txt

```diff
@@ -24,6 +25,8 @@ else()
   set(CMAKE_CXX_STANDARD 14)
   set(CXX_STANDARD_REQUIRED ON)
 
+  SET (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DTRISYCL_ASYNC=1 -fopenmp -D__TRISYCL__")
```

Comment:
Remove `-DTRISYCL_ASYNC=1`.
It does not exist anymore.
What is this `__TRISYCL__` for? When triSYCL is included, it already defines its own flags...